### PR TITLE
Update control-plane-flags.md

### PR DESCRIPTION
**Pull request by dmanbu**
Fix scheduler address binding

### CODE REVIEW

**Comment by dmanbu** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

I have completed CLA signature

File: content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ kind: ClusterConfiguration
 kubernetesVersion: v1.16.0
 scheduler:
   extraArgs:
-    address: 0.0.0.0
+    bind-address: 0.0.0.0
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md:81]: these are essentially just examples and not concrete uses cases - e.g. "setting the address for the secure port of the scheduler"

but, given the insecure port might get removed at some point the change is fine.

/lgtm
/sig cluster-lifecycle

Comment (dmanbu) [reply content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md:81]: > these are essentially just examples and not concrete uses cases - e.g. "setting the address for the secure port of the scheduler"
> 
> but, given the insecure port might get removed at some point the change is fine.
> 
> /lgtm
> /sig cluster-lifecycle

I think such an example will be misleading for beginners, such as me

**Comment by zparnold** [comment]
Hi @dmanbu we'd love to review your PR but we haven't heard from you. So I'm going to temporarily close this PR, feel free to reopen when you're ready to continue 😄 We need the check-cla: to read "yes" to continue. There are debug instructions above.
/close

**Pull request closed without merging**